src/modules/cmdpal/Microsoft.CmdPal.UI/Controls/CommandBar.xaml

```diff
@@ -259,48 +196,9 @@
                 Content="{ui:FontIcon Glyph=&#xE712;,
                                       FontSize=16}"
                 Style="{StaticResource SubtleButtonStyle}"
+                Tapped="MoreCommandsButton_Tapped"
```

Comment:
I never really know the difference between using Tapped vs. Clicked - I think it's safe to use either one, right